src/workers/cdc.ts

```diff
@@ -225,6 +227,25 @@ const onAlertsChange = async (
   }
 };
 
+const onNotificationsChange = async (
+  con: DataSource,
+  logger: FastifyLoggerInstance,
+  data: ChangeMessage<Notification>,
+): Promise<void> => {
+  if (['c'].includes(data.payload.op)) {
+    const unreadNotificationsCount = await con
+      .getRepository(Notification)
+      .count({
+        where: {
+          userId: data.payload.after.userId,
+          public: true,
+          readAt: IsNull(),
+        },
+      });
```

Comment:
Perhaps it will be better to abstract this functionality as we use it in the graphql query as well

Reply:
Ah good idea, we didn't have this before but let me refactor it 👀